rosetta/config.toml

```diff
@@ -0,0 +1,58 @@
+# GeneralSettings section of the proxy server
+[GeneralSettings]
```

Comment:
why do we have duplicated config?

Reply:
We also need rosetta config because in that config we keep the IPs of observer nodes that are in observing squad.